From 00bd3eedb8133f6f18648dbd567043a311a84c81 Mon Sep 17 00:00:00 2001
From: Jeongseok Lee <jeongseok@meta.com>
Date: Sun, 11 May 2025 16:02:49 -0700
Subject: [PATCH 1/2] Install tool binaries to the same folder of lib

---
 src/compiler/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/compiler/CMakeLists.txt b/src/compiler/CMakeLists.txt
index c885bbf2..27e3367b 100644
--- a/src/compiler/CMakeLists.txt
+++ b/src/compiler/CMakeLists.txt
@@ -198,7 +198,7 @@ function(configure_target)
 
   install(TARGETS ${target}
           LIBRARY DESTINATION lib
-          RUNTIME DESTINATION bin
+          RUNTIME DESTINATION lib
           ARCHIVE DESTINATION lib)
 endfunction(configure_target)
 
-- 
2.49.0.windows.1

